test/dtypes/test_affine_quantized_tensor_parallel.py

```diff
@@ -1,12 +1,17 @@
 from torchao.testing.utils import copy_tests, TorchAOTensorParallelTestCase
 from torch.testing._internal.common_utils import run_tests
 from torchao.quantization import int8_weight_only
+import torch
```

Comment:
do you still see the error if you move the import torch statement above any ao imports?

Reply:
Moving torch as the first import does resolve some issues, but there are still issues of compatibility with H100. I'm working on another PR to add support for H100 (specially float8). https://github.com/pytorch/ao/pull/1003